azure-pipelines.yml

```diff
@@ -205,7 +205,24 @@ stages:
           displayName: Finish background dump collection
           continueOnError: true
           condition: always()
-
+        - powershell: |
```

Comment:
@dougbu would you mind doing a quick review of the azure pipelines code here to validate our approach? It's outside of my scope would appreciate the help!